docs/concepts/cache.mdx

```diff
@@ -66,3 +71,52 @@ ory.toSession(undefined, undefined, {
   },
 })
 ```
+
+### Configuration
+
+:::note
+
+Projects on Business plan or higher can use the Session caching feature.
+
+:::
+
+```mdx-code-block
+<Tabs groupId="console-or-cli">
+  <TabItem value="console" label="Ory Console" default>
+```
+
+To configure session caching, go to **Customize** → **Advanced Settings** in the
+[Ory Console](https://console.ory.sh/projects/current/advanced-settings).
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="cli" label="Ory CLI">
+```
+
+1. Download the Ory Identities config from your project and save it to a file:
+
+```shell
+## List all available projects
+ory list projects
+
+## Get config
+ory get identity-config {project-id} --format yaml > identity-config.yaml
+```
+
+2. Add the configuration for session caching
+
+```yaml title="config.yml"
+feature_flags:
```

Comment:
Does this actually work? I remember seeing somewhere that we disabled this?

Reply:
I just tried this - pops up when you get the Identities config, throws an error when I tried to set to `true` on a non-business plan. 

```
The request was malformed or contained invalid parameters
This feature is not available on your subscription plan.
```

Reply:
Yes, it does. There's a validation to ensure only business plans or higher can do this